internal/resolver/mtaresolver_test.go

```diff
@@ -102,39 +102,45 @@ var _ = Describe("Resolve", func() {
 		wd := getTestPath("test-project")
 		yamlPath := getTestPath("test-project", "mta.yaml")
 		envGetter = mockEnvGetterWithVcapServices
-		callResolveAndValidateOutput(wd, "eb-java", yamlPath, expected)
+		callResolveAndValidateOutput(wd, "eb-java", yamlPath, expected, "")
 	})
 	It("Sanity - working dir not provided", func() {
 		yamlPath := getTestPath("test-project", "mta.yaml")
 		envGetter = mockEnvGetterExtWithVcapServices
-		callResolveAndValidateOutput("", "eb-java", yamlPath, expected)
+		callResolveAndValidateOutput("", "eb-java", yamlPath, expected, "")
 
 	})
+	It("Sanity - environment file name different from the default name (.env)", func() {
+		wd := getTestPath("test-project2")
```

Comment:
This should be changed since there's no `test-project2` folder anymore

Reply:
OK